devops/scripts/benchmarks/history.py

```diff
@@ -51,11 +56,45 @@ def extract_timestamp(file_path: Path) -> str:
             except IndexError:
                 return ""
 
+        baseline_drop_after = options.archive_baseline_days * 3
+        pr_drop_after = options.archive_pr_days * 3
+        baseline_cutoff_date = datetime.now(timezone.utc) - timedelta(
+            days=baseline_drop_after
+        )
+        log.debug(f"Baseline cutoff date: {baseline_cutoff_date}")
+        pr_cutoff_date = datetime.now(timezone.utc) - timedelta(days=pr_drop_after)
+        log.debug(f"PR cutoff date: {pr_cutoff_date}")
+
+        # Filter out files that exceed archiving criteria three times the specified days
+        def is_file_too_old(file_path: Path) -> bool:
+            try:
+                if file_path.stem.startswith("Baseline_"):
+                    cutoff_date = baseline_cutoff_date
+                else:
+                    cutoff_date = pr_cutoff_date
+
+                timestamp_str = extract_timestamp(file_path)
+                if not timestamp_str:
+                    return False
+
+                file_timestamp = datetime.strptime(timestamp_str, "%Y%m%d_%H%M%S")
```

Comment:
We now repeat the date format string a few times. Can you define it somewhere:
https://github.com/intel/llvm/pull/19514/files#diff-030b76a87d91704c2cb698ef103db9feacfbb798e990162895a31f036ad1cd75R205

Reply:
Done